sensor_msgs/Image Y0
logitech_cam/CamCmd U0
sensor_msgs/Image Y1